src/MOCK_DB_DATA/library-items/color/color-system.ts

```diff
@@ -56,6 +56,7 @@ ${colorHtml('#435c68', 'fhi-blue-grey-7')}
 <p class="text-muted small mt-2 mb-0">Blå</p>
 ${colorHtml('#0875b4', 'fhi-blue-1')}
 ${colorHtml('#393c61', 'fhi-blue-2')}
+${colorHtml('#0e4485', 'fhi-blue-3')}
```

Comment:
🤔 ...this is not correct; maybe we can do something like this:

```
${colorHtml('#0875b4', 'fhi-blue-1')}
${colorHtml('#0e4485', 'fhi-blue-3 [sic] Will be fixed in v5')}
${colorHtml('#393c61', 'fhi-blue-2')}
```

Reply:
Think it's better to do this now, since the $fhi-blue-2 and -3 are new color variables since previous release and not breaking anyway